lib/gateway/Shard.js

```diff
@@ -1137,11 +1138,12 @@ class Shard extends EventEmitter {
                 }
                 member = guild.members.update(packet.d, guild);
                 /**
-                * Fired when a member's roles or nickname are updated or they start boosting a server
+                * Fired when a member's guild avatar, roles or nickname are updated or they start boosting a server
```

Comment:
Missing `when a member passes the rules gate for a server`?

Reply:
Can also go in its own PR